src/locales/cs/translation.json

```
@@ -334,106 +334,106 @@
+            "movementPool": "Vytvořte zásobu pohybu Divokého honu zkombinováním žetonů bodování každého hráče a žetonu zavřené krčmy. Ty zamíchejte (lícem dolů)."
         },
         "skellige": {
             "playmat": [
-                "If using the playmat:",
-                "Ignore tokens, Harbor tokens are printed on the map",
-                "Place ship miniatures at the following randomized locations:"
+                "Při použití (neoprenové) herní podložky:",
+                "Ignorujte žetony přístavu, jsou již vyznačené na podložce.",
+                "Umístěte figurky lodí na níže náhodně vygenerované lokace:"
             ],
             "board": [
-                "If using the board",
-                "Shuffle the Harbor tokens, anchor side up",
-                "Place them on the coastal locations (1, 5, 6, 9, 12, 13) and flip them over",
-                "Place a ship miniature on the three locations with a ship symbol",
-                "Flip them back to the anchor side, as a reminder"
+                "Při použití klasického herního plánu:",
+                "Zamíchejte žetony přístavu, kotvou nahoru.",
+                "Umístěte je na pobřežní lokace (1, 5, 6, 9, 12, 13) a otočte je.",
+                "Umístěte figurky lodí na lokace se symbolem lodě.",
+                "Otočte žetony zpět na obrázek kotvy jako upomínku, kde jsou přístavy."
             ],
-            "decks": "Shuffle the Skellige Exploration deck. Do not shuffle the Skellige Event deck! Place both in their slots above the islands on the game board.",
-            "dagon": "Place the Dagon miniature on the starting space corresponding to player count. Shuffle the Dagon Bonus cards and place them, Dagon monster card (Special Ability face-up), and Dagon Trail tokens near its track.",
-            "dagonMT": "Place the Dagon miniature on the starting space corresponding to player count. Shuffle the Dagon Bonus cards and place them, Dagon monster card (4 Special Attacks face-up), and Dagon Trail tokens near its track.",
-            "siren": "Place the Siren miniature on the Dagon location, with its Monster Card nearby."
+            "decks": "Zamíchejte balíček průzkumu Skellige. Nemíchejte balíček událostí Skellige! Oba umístěte na jejich místo, nad ostrovy na herním plánu.",
+            "dagon": "Umístěte figurku Dagona na počáteční místo dle počtu hráčů. Zamíchejte bonusové karty Dagona a umístěte je, kartu nestvůry Dagona (speciální schopností nahoru) a žetony stopování Dagona poblíž stupnice nebezpečí.",
+            "dagonMT": "Umístěte figurku Dagona na počáteční místo dle počtu hráčů. Zamíchejte bonusové karty Dagona a umístěte je, kartu nestvůry Dagona (čtyřmi speciálními útoky nahoru) a žetony stopování Dagona poblíž stupnice nebezpečí.",
+            "siren": "Umístěte figurku Sirény na Dagonovu lokaci a poblíž její kartu nestvůry.",
```

Trailing comma needs to be removed. It causes an issue where it expects another entry but finds none and won't successfully compile.

```suggestion
            "siren": "Umístěte figurku Sirény na Dagonovu lokaci a poblíž její kartu nestvůry."
```